File: src/tool/spring-boot-generator/output/railway/db/trackDivisionInfo.js
let trackDivisionInfo = {
	name: "trackDivisionInfo",
	description: "工务段信息",
	columns: {
		bureauId: {
			name: "bureauId",
			description: "铁路局编号",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		fullName: {
			name: "fullName",
			description: "工务段名称",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		id: {
			name: "id",
			description: "编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		inputCode: {
			name: "inputCode",
			description: "输入代码",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		isDelete: {
			name: "isDelete",
			description: "删除标记",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		operator: {
			name: "operator",
			description: "操作人",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		shortName: {
			name: "shortName",
			description: "工务段简称",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		updateTime: {
			name: "updateTime",
			description: "更新时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		}
	}
};

let columnsArray = [
	"bureauId",
	"createTime",
	"fullName",
	"id",
	"inputCode",
	"isDelete",
	"operator",
	"shortName",
	"updateTime"
];

function create() {
  return {};
}

trackDivisionInfo.columnsArray=columnsArray;
trackDivisionInfo.create=create;
module.exports=trackDivisionInfo;

File: src/tool/spring-boot-generator/output/railway/railway/src/main/java/com/railway/pojo/entity/TrackDivisionInfo.java
package com.railway.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@ApiModel("工务段信息")
@Getter
@Setter
public class TrackDivisionInfo  {

    @ApiModelProperty("铁路局编号")
    private Integer bureauId;

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("工务段名称")
    private String fullName;

    @ApiModelProperty("编号")
    private Integer id;

    @ApiModelProperty("输入代码")
    private String inputCode;

    @ApiModelProperty("删除标记")
    private Integer isDelete;

    @ApiModelProperty("操作人")
    private Integer operator;

    @ApiModelProperty("工务段简称")
    private String shortName;

    @ApiModelProperty("更新时间")
    private Date updateTime;

}
